### ISSUE TITLE
Status bar handling does not work with Java Access Bridge applications

### ISSUE THREAD

**Issue by rpaquay** [nvaccess/nvda#5674]
nvda uses the bottom left corner as a reference point for finding the status bar. This does not work for Java applications because, the bottom left corner is seen as a native window control. See https://code.google.com/p/android/issues/detail?id=198428#c2 for full details.

This can easily be reproduced using the SwingSet2 demo:

* Download the SwingSet2 demo
* Apply the attached diff to SwingSet2 so that there is a real "status bar" in the app
* Run SwingSet2
* Run nvda
* use NVDA-End to speak the status bar

Result:
NVDA speaks "No status bar"

Expected:
NVDA speaks the content of the status bar "Press Shift F10 to activate pop up menu"

SwingSet2 diff: [Make-status-bar-component-a-real-a11y-STATUS_BAR.zip](https://github.com/nvaccess/nvda/files/86530/Make-status-bar-component-a-real-a11y-STATUS_BAR.zip)

**Comment by rpaquay** [comment]
Note also that some applications (IntelliJ/Android Studio) have custom status bar where the status bar text is slightly further to the right of the window, i.e. not at the very first left pixel of the client area of the window.

For reference, here is  a picture on how the status bar look in IntelliJ/Android studio:

![intellijstatusbar](https://cloud.githubusercontent.com/assets/1087507/12273612/c19ce14a-b91b-11e5-9b89-d86bc164dd1c.png)

**Comment by Adriani90** [comment]
@rpaquay seems to have some pull requests which solves issues in java applications but it seems the developer is not available anymore. Could some one look over those pull requests and rebase them to last master? ideally someone can take over those pull requests and finalize them. cc: @nvdaes, @ABuffEr, @nishimotz.

**Comment by nishimotz** [comment]
Just confirmed status bar could not found in Android Studio 3.3 via NVDA+Shift+End (laptop).
I am not sure #5675 should be used to solve this.

**Comment by Adriani90** [comment]
This could also be a candidate for custom status bars. cc: @josephsl, @JulienCochuyt

**Comment by Adriani90** [comment]
cc: @thgcode since you are familliar with JAB as well, maybe you have some suggestions here or can contribute to fixing this via custom status bar retrievals. Not sure if this is viable though.
